I.write-tests/03-remove-vowels-from-array/remove-vowels-in-array.test.js

```diff
@@ -1,9 +1,15 @@
+const removeVowelsForWords = require("./remove-vowels-in-array");
+
 test('remove vowels from all words in array', function () {
     // Arrange
+    const input = ["Irina", "Etza", "Daniel"];
```

Comment:
what happens if we add `Mikey` to this array?
If we remove the vowels we get `mky` back. Is that what is going to happen if we run the tests? ;)